src/include/migraphx/tune_axis.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_TUNE_AXIS_HPP
+#define MIGRAPHX_GUARD_OPERATORS_TUNE_AXIS_HPP
+
+#include <utility>
+#include <cstdint>
+#include <string>
+#include <migraphx/errors.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+inline int tune_axis(const int n_dim, const int axis, std::string op_name)
```

Comment:
Can we make `op_name` have default value?

Reply:
added